### PR TITLE
Show alert when on java 9

### PR DESCRIPTION
**Pull request by AustinShalit**
Possibly "solves" #397

### CODE REVIEW

File: app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java [hunk 1]
```diff
@@ -74,6 +75,27 @@ public void start(Stage primaryStage) throws IOException {
...
+    // version, show an alert and exit before we get into trouble.
+    String javaSpec = System.getProperty("java.specification.version");
+    String javaVersion = System.getProperty("java.version");
+    if (!"1.8".equals(javaSpec)) {
+      System.out.println(javaSpec);
+      Alert invalidVersionAlert = new Alert(Alert.AlertType.ERROR);
+      invalidVersionAlert.setHeaderText("Invalid JRE Version!");
+      invalidVersionAlert.setContentText(
+          String.format("You are using an unsupported Java version: %s\n"
+                  + "Please download Java 8 and uninstall Java %s.",
```
Comment (JLLeitschuh) [inline app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java]: Do they really have to uninstall their other version of java? They just need to make sure that `JAVA_HOME` is set correctly right?

Comment (AustinShalit) [reply app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java]: Certain jar launchers do not respect the java home environment variable.

Comment (JLLeitschuh) [reply app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java]: Then ship this with the Gradle Application plugin shell/command line launcher that will always respect `JAVA_HOME`.

Comment (PeterJohnson) [reply app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java]: Not an option for this year, as the DS runs the jar directly.

Comment (JLLeitschuh) [reply app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java]: So have the DS code search for java 8 and try to launch with java 8.

Comment (AustinShalit) [reply app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java]: Due to the approaching deadline of today, I think we go with this as is and circle back after season is over to make shuffleboard work with both JRE 8 & 9 or make this a better UX.

**Comment by AustinShalit** [comment]
https://github.com/wpilibsuite/shuffleboard/pull/399#discussion_r166940473

> Due to the approaching deadline of today, I think we go with this as is and circle back after season is over to make shuffleboard work with both JRE 8 & 9 or make this a better UX.

**Pull request merged**